bibformat/format_templates/Hepnames_HTML_detailed.bft

```diff
@@ -10,7 +10,7 @@
 <a href="/search?ln=en&amp;cc=HepNames&amp;ln=en&amp;cc=HepNames&amp;p=701%3A%22<BFE_FIELD tag="100__a" />%22">[Students]</a>
 <a href="http://arxiv.org/find/all/1/au:<BFE_INSPIRE_HEPNAMES_ARXIV />/0/1/0/all/0/1?per_page=100">[arXiv]</a>
 <!-- http://arxiv.org/find/all/1/au:Brooks_T/0/1/0/all/0/1?per_page=100 -->
-<a href="http://adsabs.harvard.edu/cgi-bin/author_form?author=<BFE_FIELD tag="100__q" />">[ADS]</a>
+<a href="<BFE_INSPIRE_HEPNAMES_ADS_LINK />">[ADS]</a>
```

Comment:
... or actually you might use `<BFE_INSPIRE_HEPNAMES_ADS_LINK escape=8  />`(without escaping inside the bibformat element):

See: http://invenio-demo.cern.ch/help/admin/bibformat-admin-guide#listofescapingmodes